homeassistant/components/here_weather/utils.py

```diff
@@ -0,0 +1,40 @@
+"""Utility functions for here_weather."""
+
+from homeassistant.const import CONF_UNIT_SYSTEM_METRIC
+
+
+def convert_unit_of_measurement_if_needed(unit_system, unit_of_measurement: str) -> str:
+    """Convert the unit of measurement to imperial if configured."""
+    if unit_system != CONF_UNIT_SYSTEM_METRIC:
+        if unit_of_measurement == "°C":
```

Comment:
These units of measure exist as constants in `homeassistant/const.py`. EG
`if unit_of_measurement == TEMP_CELSIUS:`